packages/terra-breakpoints/docs/README.md

```diff
@@ -0,0 +1,149 @@
+
+> Note that since the breakpoints are defined as minimum values, a breakpoint will be determined to be active if the width value is larger than the defined minimum, even if the width value is included in a higher breakpoint's range.
+
+```jsx
+import { isBreakpointActiveForSize } from 'terra-breakpoints';
+
+console.log(isBreakpointActiveForSize('tiny', 300));      // true
+console.log(isBreakpointActiveForSize('tiny', 544));      // true
+console.log(isBreakpointActiveForSize('tiny', 800));      // true
+console.log(isBreakpointActiveForSize('tiny', 1000));     // true
+
+console.log(isBreakpointActiveForSize('medium', 300));    // false
+console.log(isBreakpointActiveForSize('medium', 544));    // false
+console.log(isBreakpointActiveForSize('medium', 800));    // true
+console.log(isBreakpointActiveForSize('medium', 1500));   // true
+
+console.log(isBreakpointActiveForSize('enormous', 300));  // false
+console.log(isBreakpointActiveForSize('enormous', 544));  // false
+console.log(isBreakpointActiveForSize('enormous', 800));  // false
+console.log(isBreakpointActiveForSize('enormous', 1500)); // true
+```
+
+### ActiveBreakpointProvider
+
+`terra-breakpoints` also defines a component that provides the current active breakpoint to child components using a Context-based API. This can be used to minimize the number of required resize event listeners and ensure consistency across the component hierarchy.
+
+A component generator called `withActiveBreakpoint()` is included to provide simple interfacing with the ActiveBreakpointProvider. However, the ActiveBreakpointContext can also be imported and used directly, if needed.
+
+```jsx
+import React from 'react';
+import { ActiveBreakpointProvider, withActiveBreakpoint, ActiveBreakpointContext } from 'terra-breakpoints';
+
+const ActiveBreakpointConsumer1 = withActiveBreakpoint(({ activeBreakpoint }) => (
+  <div>
+    <p>This component uses the withActiveBreakpoint() component generator to interface with the ActiveBreakpointProvider.</p>
+    <p>
+      The active breakpoint is:
+      {' '}
+      {activeBreakpoint}
+    </p>
+  </div>
+));
+
+const ActiveBreakpointConsumer2 = () => (
+  <ActiveBreakpointContext.Consumer>
+    {
+      activeBreakpoint => (
+        <div>
+          <p>This component interfaces with the ActiveBreakpointContext directly.</p>
+          <p>
+            The active breakpoint is:
+            {' '}
+            {activeBreakpoint}
+          </p>
+        </div>
+      )
+    }
+  </ActiveBreakpointContext.Consumer>
+);
+
+const ActiveBreakpointProviderExample = () => (
+  <ActiveBreakpointProvider>
+    <ActiveBreakpointConsumer1 />
+    <hr />
+    <ActiveBreakpointConsumer2 />
+  </ActiveBreakpointProvider>
+);
+```
+
+### Media Queries
+
+`terra-breakpoints` includes `_media-queries.scss`, which contains a set of SASS mixins that define media queries for the supported breakpoints.
```

Comment:
minor nit: `Sass` instead of `SASS`. http://sassnotsass.com/

Reply:
![https://media.giphy.com/media/MoYC1N4nv7Fcs/giphy.gif](https://media.giphy.com/media/MoYC1N4nv7Fcs/giphy.gif)